src/analysis/classic/deathknight/blood/CombatLogParser.ts

```diff
@@ -6,6 +6,7 @@ import AlwaysBeCasting from './modules/features/AlwaysBeCasting';
 import Buffs from './modules/features/Buffs';
 import CooldownThroughputTracker from './modules/features/CooldownThroughputTracker';
 import FoundationGuide from 'interface/guide/foundation/FoundationGuide';
+import ArmyOfTheDead from '../shared/ArmyOfTheDead';
 // Spells
 // import SpellName from './modules/spells';
```

Comment:
Remove import comment now that there is a spell. Also, the `// Spells` comment can be moved above line 9